packages/utils/src/lib/reports/__snapshots__/log-stdout-summary.integration.test.ts.snap

```diff
@@ -72,24 +72,21 @@ Lighthouse audits
 ● Cumulative Layout Shift                                             0
 ● Total Blocking Time                                                 0 ms
 
-
 Categories
 
-┌─────────────────────────────────────────────────────────────┬───────┬────────┐
-│ Category                                                    │ Score │ Audits │
-├─────────────────────────────────────────────────────────────┼───────┼────────┤
-│ Performance                                                 │    92 │      8 │
-├─────────────────────────────────────────────────────────────┼───────┼────────┤
-│ Bug prevention                                              │    68 │     16 │
-├─────────────────────────────────────────────────────────────┼───────┼────────┤
-│ Code style                                                  │    54 │     13 │
-└─────────────────────────────────────────────────────────────┴───────┴────────┘
-
-Made with ❤ by code-pushup.dev
-"
+┌─────────────────────────────────────────────────────────┬─────────┬──────────┐
+│  Category                                               │  Score  │  Audits  │
+├─────────────────────────────────────────────────────────┼─────────┼──────────┤
+│  Performance                                            │     92  │       8  │
+├─────────────────────────────────────────────────────────┼─────────┼──────────┤
+│  Bug prevention                                         │     68  │      16  │
+├─────────────────────────────────────────────────────────┼─────────┼──────────┤
+│  Code style                                             │     54  │      13  │
+└─────────────────────────────────────────────────────────┴─────────┴──────────┘
+Made with ❤ by code-pushup.dev"
 `;
```

Comment:
Seems like we lost some separator lines. I think the original was better spaced out, ideally we would preserve that.

Reply:
I was not able to get it working... there is a different layouting logic in the background now..

This is what happens when I try to make it identical...

<img width="155" alt="Screenshot 2024-03-12 at 12 17 40" src="https://github.com/code-pushup/cli/assets/10064416/517a48a9-ee6e-420e-9aa4-99e68ee87997">

Reply:
But the lines aren't related to be table. Why not just add some `log()` calls in `log-stdout-summary.ts` to create those extra lines?

Reply:
Aaa i added it! Maybe a empty string is the case. Sry I missed this.

Reply:
space is back!

Reply:
Almost there :sweat_smile: Just the footer is missing the separator lines now:

![image](https://github.com/code-pushup/cli/assets/34691111/3a1ce62d-53ec-499d-ac7e-d3af85a611e8)